data/coalition/coalition weapons.txt

```diff
@@ -111,6 +111,8 @@ outfit "Finisher Pod"
 		"homing" 2
 		"radar tracking" .8
 		"missile strength" 50
+		"damage dropoff" 1 1
+		"dropoff modifier" 0.85
```

Comment:
This isn't really a good use of the damage dropoff mechanic in my mind. If your desire is "the parent projectile deals less damage than the child submunition," then I'd suggest doing what is done with the Korath Minelayer. That is, the parent projectile deals negative damage, so that if the parent projectile hits something before splitting it deals less damage than if it had split.